.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 language: go
 go_import_path: github.com/redhat-cop/quay-operator
 go:
-- "1.12"
+  - "1.13"
```

Comment:
Is golang 1.13 required to build? The main reason I ask is because Redhat's OSBS/build pipeline is still using 1.12.

Reply:
Yes. it is now required by operator-sdk 0.12